app/src/main/java/eu/kanade/tachiyomi/ui/reader/viewer/pager/PagerConfig.kt

```diff
@@ -29,6 +29,10 @@ class PagerConfig(
     var imageCropBorders = false
         private set
 
+    // SY -->
+    var usePageTransitionsPager = false
```

Comment:
Maybe keep the name usePageTransitions so we dont have to change the rest of the code